packages/dnb-eufemia/src/components/list-format/ListFormat.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useContext, useMemo } from 'react'
+import { LOCALE } from '../../shared/defaults'
+import {
+  convertJsxToString,
+  extendPropsWithContext,
+} from '../../shared/component-helper'
+import SharedContext, { InternalLocale } from '../../shared/Context'
+import { Li, Ol, Ul } from '../../elements'
+
+export type ListFormatProps = {
+  /**
+   * Formatting options for the value.
+   * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat
+   */
+  format?: Intl.ListFormatOptions
+  /**
+   * Defines if the value should be displayed in list format or regular text format on one line.
+   * Default: `text`
+   */
+  variant?: 'ol' | 'ul' | 'text'
+  /**
+   * Defines the type of list styling used for list variants. Used together with variant `ol` and `ul`.
+   * Variant `ol`: `a`, `A`, `i`, `I` and `1`.
+   * Variant `ul`: `circle`, `disc` and `square`.
+   * Default: `undefined`
+   */
+  listType?:
+    | 'a'
+    | 'A'
+    | 'i'
+    | 'I'
+    | '1'
+    | 'circle'
+    | 'disc'
+    | 'square'
+    | undefined
+
+  /**
+   * The value to format as list.
+   * Default: null
+   */
+  value?: Array<React.ReactNode>
+
+  /**
+   * The children to format as list.
+   * Default: null
+   */
+  children?: React.ReactNode
+}
+
+function ListFormat(localProps: ListFormatProps) {
+  const { locale, ListFormat } = useContext(SharedContext)
+
+  // Extract additional props from global context
+  const allProps = extendPropsWithContext(localProps, {}, ListFormat)
+  const { value, format, variant = 'text', listType, children } = allProps
+
+  const list = useMemo(() => {
+    const isListVariant = variant !== 'text'
+    if (children) {
+      return isListVariant
+        ? React.Children.map(children, (child: React.ReactNode, index) => {
+            return <Li key={index}>{child}</Li>
+          })
+        : children
+    }
```

Comment:
```suggestion
const list = value || children
```

I think we should handle `value` and `children` the same. So just create a new

Reply:
Done :)